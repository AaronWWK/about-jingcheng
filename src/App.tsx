import React, { useState } from 'react';
import 'semantic-ui-css/semantic.min.css'
import { SideNavigationBar } from 'components/sideNavigationBar/SideNavigationBar';

const App = () => {
  const [sideBarVisible, setSidebarVisible] = useState(false);
  // Update side bar visibility to show animation
  setTimeout(() => setSidebarVisible(true));
  return (
    <div className="App">
      <SideNavigationBar visible={sideBarVisible}/>
    </div>
  );
}

export default App;
