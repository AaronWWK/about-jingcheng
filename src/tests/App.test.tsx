import React from 'react';
import { shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';
import App from 'App';

test('renders correctly', () => {
  const component = shallow(<App />);
  expect(toJSON(component)).toMatchSnapshot();
});
