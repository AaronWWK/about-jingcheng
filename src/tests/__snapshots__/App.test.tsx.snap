// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<HashRouter>
  <SidebarPushable
    as={[Function]}
  >
    <SideNavigationBar />
    <SidebarPusher>
      <MainContent
        sideBarReady={false}
      />
    </SidebarPusher>
  </SidebarPushable>
</HashRouter>
`;
